lib/ex_doc/markdown.ex

```diff
@@ -8,9 +8,9 @@ defmodule ExDoc.Markdown do
   * [Earmark][]
   * [Pandoc][]
 
-  If you don't offer any preference via `config/config.exs`. ExDoc will try to
-  find one of the earlier MarkDown parsers. Otherwise, ExDoc will raise an
-  exception.
+  If you don't specify a parser in `config/config.exs`. ExDoc will try to
+  find one of the MarkDown parser from list in top down fashion. Otherwise, 
```

Comment:
Seems to be missing a word around "list". This sentence reads odd: "ExDoc will try to find one of the MarkDown parser from list in top down fashion.".